src/components/scroll-links.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { Element } from 'react-scroll'
+
+export default function () {
+  return (
+    <React.Fragment>
+
+      <Element id='home' name='home'>
+        // wrap your content in the Element from react-scroll
```

Comment:
This should technically take children as a prop and then return children in the element, can you add a link to where you found this implementation?

Reply:
This file specifically is going to be deleted. I was trying out this implementation directly from a blog I was reading but didn't have a chance to determine for sure whether I needed it or not. I just deleted the file on my local copy and it is not needed so I'll remove that from the request. 

The Element component is the only thing that has been used in each individual section.